src/mds/Mutation.h

```diff
@@ -158,8 +158,8 @@ struct MutationImpl : public TrackedOp {
     return lock == last_locked;
   }
 
-  bool is_master() const { return slave_to_mds == MDS_RANK_NONE; }
```

Comment:
In this context, would `is_authority` be clearer?

Reply:
I don't think so